collectors/githubreceiver/kustomization.yaml

```diff
@@ -0,0 +1,29 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: collector
+
+resources:
+  - ../base/
+
+generatorOptions:
+  disableNameSuffixHash: true
+
+secretGenerator:
+  - name: git-pat
```

Comment:
Referring to my second comment. Make sure this is also changed to github-pat, else it will fail